File: Basic_Math/prime_number/primeNumber.py
def isPrime(n):
    if ( countFactor(n) == 2):
        return True
    return False

def countFactor(n):
    count = 0
    for i in range(1, int(n**0.5)+1):
        if ( n % i == 0 ):
            if ( n//i == i):
                count += 1
            else:
                count += 2
    return count

print("is prime: ",isPrime(2))

File: Basic_Math/prime_number/primeNumber.js
const isPrime  = ( n ) =>  {
    if ( countFactor(n) === 2 ) return true;
    return false;
}

const countFactor = ( n ) => {
    let count = 0;
    for ( let i = 0 ; i * i <= n ; i++ ) {
        if ( n % i === 0 ) {
            if ( n / i === i ) {
                count++;
            } else {
                count += 2;
            }
        }
    }
    return count;
}

console.log("is prime: ",isPrime(2));